ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -734,6 +734,11 @@ public void setNumLines(ReactEditText view, int numLines) {
     view.setLines(numLines);
   }
 
+  @ReactProp(name = ViewProps.MAXIMUM_NUMBER_OF_LINES, defaultInt = 1)
```

Comment:
Isn't it better to have `-1` (UNSET) as default?

Reply:
Yes, I'm rewriting android part now as I found few issues with current implementation. Hope to finish it today.